From 7ae4ddec14a4bf58536df7d5ac4d8ca0587e1952 Mon Sep 17 00:00:00 2001
From: Salini Venate <salini.venate@intel.com>
Date: Tue, 27 Jun 2023 05:44:39 +0530
Subject: [PATCH 1/3] Comment the ndk version

Comment out ndk version for now. This patch will be reverted
after aosp upgrade.

Tracked-On: None
Signed-off-by: Salini Venate <salini.venate@intel.com>
---
 groups/device-specific/celadon_ivi/compatibility_matrix.xml | 4 ++--
 groups/device-specific/celadon_ivi/framework_manifest.xml   | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/groups/device-specific/celadon_ivi/compatibility_matrix.xml b/groups/device-specific/celadon_ivi/compatibility_matrix.xml
index 4389dac..4221809 100644
--- a/groups/device-specific/celadon_ivi/compatibility_matrix.xml
+++ b/groups/device-specific/celadon_ivi/compatibility_matrix.xml
@@ -47,7 +47,7 @@
             <instance>default</instance>
         </interface>
     </hal>
-    <vendor-ndk>
+    <!--<vendor-ndk>
         <version>33</version>
-    </vendor-ndk>
+    </vendor-ndk>-->
 </compatibility-matrix>
diff --git a/groups/device-specific/celadon_ivi/framework_manifest.xml b/groups/device-specific/celadon_ivi/framework_manifest.xml
index d33abc0..acbd3c2 100644
--- a/groups/device-specific/celadon_ivi/framework_manifest.xml
+++ b/groups/device-specific/celadon_ivi/framework_manifest.xml
@@ -193,7 +193,7 @@
             <instance>default</instance>
         </interface>
     </hal>
-    <vendor-ndk>
+    <!--<vendor-ndk>
         <version>33</version>
-    </vendor-ndk>
+    </vendor-ndk>-->
 </manifest>
-- 
2.17.1

